Modules/Bridge/VtkGlue/CMakeLists.txt

```diff
@@ -46,6 +46,8 @@ endif()
 set(_target_prefix "vtk")
 if(VTK_VERSION VERSION_GREATER_EQUAL 8.90.0)
   set(_target_prefix "VTK::")
+  message(STATUS "VTK supports image direction")
+  set(VTK_SUPPORTS_DIRECTION 1) #definition used in itkVtkGlueConfigure.h.in
```

Comment:
It would be better to use VTK version C preprocessor macros defined in *vtkVersion.h* instead of CMake variables. Although we could technically say that we do not support it, someone may reasonably try to use the ITKVtkGlue headers in a project where they use a different VTK than the version that ITKVtkGlue was built with. Also, it keeps things simpler and more direct in the implementation. For example, there is a bugs related to the complexity right now, i.e. *itkVtkGlueConfigure.h* is not installed.

Reply:
Now that i see the use case, what about adding `VTK_SUPPORT_DIRECTION` directly in `vtkConfigure.h` (or `vtkCommonDataModelConfigure.h`)

Reply:
You are correct because the value of this macro is not used inside any `cxx` file build as part of this library. If it were the case, then for consistency we would want to define that extra macro.

Reply:
@thewtex updates as recommended. Current use is:
```c++
#if VTK_MAJOR_VERSION >= 9 || (VTK_MAJOR_VERSION == 8 && VTK_MINOR_VERSION >= 90)
```
but we could update to just this in the future:
```c++
#if VTK_MAJOR_VERSION >= 9
```

Let me know if we want to consider adding another preprocessor definition in VTK (but maybe it becomes confusing to have that many, and checking the version is easier for users?)

Reply:
Direction support being a major addition, i think we should add a proper macro in VTK.